engine-nextgen/src/main/kotlin/graphql/nadel/validation/util/NadelInputValidationUtil.kt

```diff
@@ -0,0 +1,85 @@
+package graphql.nadel.validation.util
+
+import graphql.nadel.validation.util.NadelSchemaUtil.getUnderlyingName
+import graphql.schema.GraphQLEnumType
+import graphql.schema.GraphQLInputObjectType
+import graphql.schema.GraphQLInputType
+import graphql.schema.GraphQLNamedType
+import graphql.schema.GraphQLNonNull
+import graphql.schema.GraphQLScalarType
+import graphql.schema.GraphQLTypeUtil
+
+object NadelInputValidationUtil {
```

Comment:
Can you copy the code in `NadelTypeValidation.isOutputTypeValid` and flip the logic instead?